### PR TITLE
Theme setup

### PR DESCRIPTION
**Pull request by Azein**
Added default theme on top of @oyabun existing specs
Reworked a bit the utils with intention to simplify the API.
Stories now live together with components

To be done still - check our application-level import opportunities.

### CODE REVIEW

File: src/media.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const devices = {
```
Comment (VLegakis) [inline src/media.ts]: Should @oyabun confirm these breakpoints?

File: src/theme/utils.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import math from "polished/lib/math/math"
+import { mergeDeepRight, path, pathOr } from "ramda"
+
+export type ThemeAtom = string | number | { [key: string]: string | number | ThemeAtom }
```
Comment (VLegakis) [inline src/theme/utils.ts]: Why does this Theme need to opt between all these types?
Wouldn't just a (simple) Theme interface do fine?

Comment (builat) [reply src/theme/utils.ts]: @VLegakis coz, there are could be sublevels. Thishow recursive typing implemented here

Comment (Azein) [reply src/theme/utils.ts]: Theme could be constructed on application level, with keys/values we have no idea about right now. So, we use a wider type, also setting some constraints.

File: src/media.ts (hunk 1 above)
Comment (Azein) [reply src/media.ts]: He did.

**Pull request merged**